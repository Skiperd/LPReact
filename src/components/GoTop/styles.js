import styled,{css} from 'styled-components';
import {theme } from '../../styles/theme'

export const Container = styled.a`
  ${({theme}) => css`
    position: fixed;
    background: ${theme.colors.primaryColor};
    color: ${theme.colors.white};
    display: flex;
    align-items: center;
    justify-content: center;
    width: 4rem;
    height: 4rem;
    bottom: 2rem;
    right: 2rem;
    z-index: 6;
    /* opacity: 0.5; */
    cursor: pointer;
    border-radius: 10px;
  `}
`;
