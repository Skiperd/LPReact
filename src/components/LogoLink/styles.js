import styled,{css} from 'styled-components';
import {theme } from '../../styles/theme'
export const Container = styled.a`
  ${({theme}) => css`
    text-decoration: none;
    align-items: center;
    color: inherit;

    > img {
      max-height: 3rem;
    }
  `}
`;
