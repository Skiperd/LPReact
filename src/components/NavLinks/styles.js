import styled,{css} from 'styled-components';
import { theme } from '../../styles/theme'
export const Container = styled.nav`
  ${({theme}) => css`
    display: flex;
    flex-flow: row wrap;

    @media ${theme.media.lteMedium} {
      flex-flow: column wrap;
      align-content: center;
    }
  `}
`;
