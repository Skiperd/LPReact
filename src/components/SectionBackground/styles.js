import styled,{css} from 'styled-components';
import {theme } from '../../styles/theme'

const containerBackgroundActivate = () => css`
  background: ${theme.colors.primaryColor};
  color: ${theme.colors.white};
`

export const Container = styled.div`
  ${({theme, background}) => css`
    background: ${theme.colors.white};
    color: ${theme.colors.primaryColor};
    ${background && containerBackgroundActivate(theme)};
    min-height: 100vh;
    display: flex;
    align-items: center;
  `}
`;

