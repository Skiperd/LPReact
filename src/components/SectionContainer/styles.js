import styled,{css} from 'styled-components';
import {theme } from '../../styles/theme'
export const Container = styled.div`
  ${({theme}) => css`
    max-width: 1200px;
    margin: 0 auto;
    padding: ${theme.spacings.large}
  `}
`;
