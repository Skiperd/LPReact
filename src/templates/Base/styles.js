import styled,{css} from 'styled-components';
import {theme } from '../../styles/theme'
import {Container as TextComponent} from '../../components/TextComponent/styles'

export const Container = styled.div`
  ${({theme}) => css`
  /* padding-top: 5.4rem;

  @media ${theme.media.lteMedium} {
    padding-top: 0;
  } */
  `}
`;
