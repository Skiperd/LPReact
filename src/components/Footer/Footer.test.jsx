import { renderTheme } from "../../styles/render-theme";
import { Footer } from ".";
import { screen } from "@testing-library/react";
describe("<Footer />", () => {
  it("should render", () => {
    const { container } = renderTheme(<Footer html={"<h1>Ola</h1>"} />);
    expect(screen.getByRole("heading")).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
