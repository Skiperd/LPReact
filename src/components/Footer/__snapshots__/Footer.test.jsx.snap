// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render 1`] = `
<div>
  <footer
    class="sc-dkzCjb coIidp"
  >
    <div
      class="sc-gsnScm iuPRvZ"
    >
      <div
        class="sc-bczSft bzujiZ"
      >
        <h1>
          Ola
        </h1>
      </div>
    </div>
  </footer>
</div>
`;
